.github/workflows/cicd-pipeline.yaml

```diff
@@ -2,10 +2,15 @@ name: CI/CD Pipeline
 
 on:
   push:
-  pull_request:
     branches:
-      - main
-    types: [opened, synchronize, reopened]
+      - main-jp
```

Comment:
Should this be main?

Reply:
Good spot! But its intentional so this pipeline doesn't fire. Once we are happy with the new pipelines that replace this one, we can delete it.